tests/test_rss_response.py

```diff
@@ -8,7 +8,7 @@ def test_rss_sample_response(client: TestClient, expected_response: str, capsys)
     response = client.get('/1')
     assert response.status_code == 200
     tree = etree.fromstring(response.content)
-    pretty: str = etree.tostring(tree, pretty_print=True, xml_declaration=True).decode('ascii')
+    pretty: str = etree.tostring(tree, pretty_print=True, xml_declaration=True).decode('utf-8')
```

Comment:
With my other comment, this change is not needed anymore, instead make a new test specifically for the utf8 encoded version.